FoodFriends/app/src/main/java/com/example/foodfriends/misc/RestaurantServer.java

```diff
@@ -156,7 +162,8 @@ public void findRestaurants(String apiKey) {
             } catch (ParseException e) {
                 e.printStackTrace();
             }
-            /*
+
+             */
             query.findInBackground(new FindCallback<Restaurant>() {
```

Comment:
love how u are using callback mechanism. What is generally the delay to get the restaurant list with this api.

Reply:
1-2 seconds to get the restaurants from Yelp, so pretty quick!